src/Pawn.py

```diff
@@ -102,13 +117,18 @@ def getPossibleMoves(self) -> Iterator[Move]:
                    lastMove.newPos - lastMove.oldPos == C(0, -2):
                     lastMoveWasAdvanceTwo = True
 
-            if pieceBesidePawn and \
-               pieceBesidePawn.stringRep == Pawn.stringRep and \
-               pieceBesidePawn.side != self.side and \
-               lastPieceMoved is pieceBesidePawn and \
-               lastMoveWasAdvanceTwo:
-                move = Move(self, self.position + movement,
-                            pieceToCapture=pieceBesidePawn)
+            if (
+                pieceBesidePawn
+                and pieceBesidePawn.stringRep == Pawn.stringRep
+                and pieceBesidePawn.side != self.side
+                and lastPieceMoved is pieceBesidePawn
+                and lastMoveWasAdvanceTwo
```

Comment:
I think we discussed this before. Please indent this correct.

Reply:
```
            if pieceBesidePawn \
                    and pieceBesidePawn.stringRep == Pawn.stringRep \
                    and pieceBesidePawn.side != self.side \
                    and lastPieceMoved is pieceBesidePawn \
                    and lastMoveWasAdvanceTwo:
```
?